src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -89,10 +91,10 @@ public async Task<ActionResult<OrderModel>> GetAsync(string orderId)
                     Address = order.SupplierAddress.ToModel(),
                     PrimaryContact = order.SupplierContact.ToModel()
                 },
-                TotalOneOffCost = order.CalculateCostPerYear(CostType.OneOff),
+                TotalOneOffCost = totalOneOffCost,
                 TotalRecurringCostPerMonth = calculatedCostPerYear / monthsPerYear,
                 TotalRecurringCostPerYear = calculatedCostPerYear,
-                TotalOwnershipCost = 0m,
+                TotalOwnershipCost = totalOneOffCost + (defaultContractLength * calculatedCostPerYear),
```

Comment:
Here I am questioning whether or not the view should know about the default contract length. I am thinking it is more business logic. Probably a little strange seen as you have already got the other costs but would like to propose moving this logic into a method in the Order class. What do you think? It would also use to add unit tests to check the method as well. If another part of the application needed this information it could just ask the order & potentially not have to repeat the same bit of logic.

Reply:
Good suggestion, I'll change that round